src/remote/ssh.rs

```
@@ -0,0 +1,768 @@
+        self
+    }
+
+    pub fn destination(&self) -> String {
+        let host_str = self.host.to_string();
+        match &self.username {
+            Some(username) => format!("{}@{}", username, host_str),
+            None => host_str,
+        }
+    }
+
+    pub fn connection_key(&self) -> String {
```

Clippy (1.95.0)

[warning] method `connection_key` is never used

(warning)